translations/story/04/1008/005 (上滑りのcool).json

```diff
@@ -0,0 +1,339 @@
+{
+    "version": 3,
+    "bundle": "IMAKZCFUMT6QLMJ55QCGBFJQS2YTC6NF",
+    "type": "story",
+    "storyId": "041008005",
+    "title": "上滑りのcool",
+    "text": [
+        {
+            "jpName": "",
+            "enName": "",
+            "jpText": "トレーニングコースを歩いていると……\r\n別のウマ娘と向き合っているウオッカの姿を見かけた。",
+            "enText": "While walking around the training course, \nI stumbled upon Vodka talking to another horsegirl.",
+            "nextBlock": 2,
+            "pathId": -721303237881982302,
+            "blockIdx": 1
+        },
+        {
+            "jpName": "ウオッカ",
+            "enName": "",
+            "jpText": "同じウマ娘なら――格上も格下もねぇ。\r\n正々堂々、ハンデなしで勝負だっ、先輩！！",
+            "enText": "We're both horsegirls, so we're on equal footing. \nLet's have a fair and square race with no handicaps!",
+            "nextBlock": 3,
+            "pathId": -3731656645876264917,
+            "blockIdx": 2
+        },
+        {
+            "jpName": "同級ウマ娘",
+            "enName": "",
+            "jpText": "ウオッカのやつ、まあた先輩相手に勝負挑んでるよ……。\r\nどうなることやら……。",
+            "enText": "Looks like Vodka's challenging an upperclassmen again… \nI wonder how that'll turn out for her…",
```

Comment:
Singular/plural mix: `an upperclassmen` (also upperclasswoman might be more fitting)